recipes/prince/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "prince" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f6dc02fa63bf2b1b0a5edb7c1f30efc062e498bbf705a207b8bb5e1fa3fec4fa
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py2k]
+  
+requirements:
+  host:
+    - matplotlib
```

Comment:
Do you need all these dependencies to install the package or just ``python``, ``setuptools`` and ``pip``?